parentheses/service/service.go

```diff
@@ -0,0 +1,58 @@
+// Package service implements a parentheses web service that
+// generates a random sequence of parentheses of the given length.
+package service
+
+import (
+	"errors"
+	"log"
+	"math/rand"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+// Package level Error.
+var ErrIncorrectLength = errors.New("incorrect length when you need a positive number")
+
+const bracketsArray = "(){}[]"
+
+// processRequest accepts a query with length parameter and
+// returns the random generated string of brackets with specified length.
+func processRequest(w http.ResponseWriter, r *http.Request) {
+	n := r.URL.Query().Get("n")
+
+	length, err := strconv.Atoi(n)
+	if err != nil || length <= 0 {
+		http.Error(w, ErrIncorrectLength.Error(), http.StatusBadRequest)
+	}
+
+	resultStr, err := generateRandomSequence(length)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	}
```

Comment:
This code will write the error twice for an invalid `n`

Reply:
> This code will write the error twice for an invalid `n`

Sorry, I was in a hurry to push